app/src/main/res/layout/dialog_fragment_date_time_reservation.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        tools:ignore="MissingConstraints">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+
+            <DatePicker
+                android:id="@+id/date_picker_dialog_fragment"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:datePickerMode="calendar"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:layout_editor_absoluteX="69dp" />
```

Comment:
ohh nooo, a hardcode! ajaj no matter, please remove

Reply:
it is a tools property, not an android one, why do you need this tools property?

Reply:
I want to align to center manual, but now I check it and it´s not necessary. Deleted.